core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -425,8 +426,26 @@ private void tryStart(Instant startedAt) {
             // For all registered output consumers, start following as close to container startup as possible
             this.logConsumers.forEach(this::followOutput);
 
+            // Wait until inspect container returns the mapped ports
+            if (exposedPorts.size() > 0) {
```

Comment:
Could we do without this outer `if`? It seems to me that the the branches are identical in their effects, ultimately.

Reply:
You are right.